### ISSUE TITLE
Decreased circulating C1-esterase inhibitor concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7410]
**Preferred term label:**
Decreased circulating C1-esterase inhibitor concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Decreased concentration of C1-esterase inhibitor (C1INH) in the blood circulation. 

Comment:  C1-esterase inhibitor (C1INH) is an acute phase reactant and a serine protease inhibitor (serpin). Serpins inhibit the steps in the lectin complement pathway as well as kinin-producing pathways. While C1 inhibitor has many functions, its function in the kinin pathway leads to angioedema ctivated factor XII of the coagulation cascade and kallikrein catalyze the cleavage of kininogen, which produces bradykinin. C1 inhibitor usually limits this bradykinin production by inhibiting kallikrein and factor XIIa. Bradykinin is a potent vasodilator due to multifactorial downstream effects on the smooth muscle in blood vessels.
PMID: 30844193

**Parent term (use hpo.jax.org/app)**
[Abnormality of complement system HP:0005339](https://hpo.jax.org/app/browse/term/HP:0005339)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
